sources/packages/api/src/route-controllers/application/application.system.controller.ts

```diff
@@ -42,9 +44,9 @@ import { IUserToken } from "../../auth/userToken.interface";
  * to process and have access to all data as needed.
  */
 @AllowAuthorizedParty(AuthorizedParties.formsFlowBPM)
-@Controller("system-access/application")
-@ApiTags("system-access")
-export class ApplicationSystemController extends BaseController {
+@Controller("application")
+@ApiTags(`${ClientTypeBaseRoute.SystemAccess}-application`)
+export class ApplicationSystemAccessController extends BaseController {
```

Comment:
Please rename the file accordingly to the client name `SystemAccess`.

Reply:
oh..i missed it updated